src/components/ContactCircle.tsx

```diff
@@ -55,7 +55,10 @@ function ContactCircle({
       return (
         <Text
           allowFontScaling={false}
-          style={[typeScale.labelMedium, { fontSize: iconSize / 2.0, color: fontColor }]}
+          style={[
+            typeScale.labelMedium,
+            { fontSize: iconSize / 2.0, color: fontColor, lineHeight: iconSize / 1.5 },
```

Comment:
There was an issue here with the capital letter in the avatar being off-center and cut off. Not sure if this was related to the recent port over to using `typeScale`.